src/lib/OpenEXR/ImfMisc.cpp

```diff
@@ -288,7 +289,7 @@ copyIntoFrameBuffer (
             {
                 unsigned int fillVal = (unsigned int) (fillValue);
 
-                while (writePtr <= endPtr)
+                while (writePtr != endPtr)
```

Comment:
The less than checks were originally a safeguard against overruns and non-termination. With an exact test, I think it would be important to do a pre-check at the start of the function to ensure that endPtr can actually be exactly met given start, end, and stride.

Reply:
OK, I added some `assert`